_data/events/Paris_.json

```diff
@@ -0,0 +1,17 @@
+{
+  "title": "GDCR17 in Paris by Nexeo",
+  "url": "https://global-day-of-code-retreat-by-nexeo.eventbrite.com",
+  "moderators": [
+    "@NexeoFinance"
```

Comment:
Please add the actual moderators here, not (just) the company sponsoring the event.